test/e2e/example_k8petstore.go

```diff
@@ -150,7 +150,7 @@ T:
 	Ω(totalTransactions).Should(BeNumerically(">", minExpected))
 }
 
-var _ = Describe("[Example] Pet Store [Skipped]", func() {
+var _ = Describe("Pet Store [Feature:Example]", func() {
```

Comment:
this will now make these tests run ~everywhere. I don't think that's what we want?